uberfire-workbench/uberfire-workbench-client/src/main/java/org/uberfire/client/mvp/LockManagerImpl.java

```diff
@@ -309,6 +321,12 @@ void onSaveInProgress(@Observes SaveInProgressEvent evt) {
         }
     }
 
+    void onRenameInProgress(@Observes RenameInProgressEvent event) {
+        if (getLockTarget() != null && event.getPath().equals(lockTarget.getPath())) {
```

Comment:
... here you're OK using ```Path``` equality? (not URI, like your change above). Is this OK?

Reply:
...here, we _always_ have an `ObservablePath` being compared to another `ObservablePath`.

I added a commentary to make this less confusing.

Thanks for the comment :-)